modules/compliance-tailored-profiles.adoc

```diff
@@ -74,15 +72,24 @@ spec:
 |`setValues`
 | A list of name, rationale, and value groupings. Each name refers to a name of the value set. The rationale is human-readable text describing the set. The value is the actual setting.
 |===
-+
-. Add the profile to the `ScanSettingsBinding` object:
+
+.. Create the `TailoredProfile` object:
 +
 [source,terminal]
 ----
-$ cat nist-moderate-modified.yaml
+$ oc create -n openshift-compliance -f new-profile-node.yaml <1>
 ----
+<1> The `TailoredProfile` object is created in the default `openshift-compliance` namespace.
 +
 .Example output
+[source,terminal]
+----
+tailoredprofile.compliance.openshift.io/nist-moderate-modified created
```

Comment:
@kmccarron-rh Hi, please add description in above `tailoredprofile` otherwise it fails on description

Reply:
Also the rule `rhcos4-file-permissions-node-config` looks invalid so you can add any valid rule e.g `rhcos4-file-permissions-var-log-messages` instead 

> apiVersion: compliance.openshift.io/v1alpha1
> kind: TailoredProfile
> metadata:
>   name: nist-moderate-modified
> spec:
>   extends: rhcos4-moderate
>   description: NIST moderate profile
>   title: My modified NIST moderate profile
>   disableRules:
>   - name: rhcos4-file-permissions-var-log-messages
>     rationale: The file contains logs of error messages in the system
>   - name: rhcos4-account-disable-post-pw-expiration
>     rationale: No need to check this as it comes from the IdP
>   setValues:
>   - name: rhcos4-var-selinux-state
>     rationale: Organizational requirements
>     value: permissive
>

Reply:
This is done.